index.html

```diff
@@ -171,8 +171,27 @@ <h2>Scope</h2>
           It is explicitly not a requirement that the new specifications be fully compatible with related past specifications.
         </p>
         <p>
-          Once all the planned Recommendations have been published, the Working Group will continue in maintenance mode to handle errata. No new Recommendations are planned. <a href="https://www.w3.org/2023/Process-20231103/#class-4">Class 4 changes</a> for Recommendations published by the Working Group are <strong>out of scope</strong>, <em><strong>except</strong></em> if serious security issues come to the fore that require changes in a Recommendation.
+          Once all the planned Recommendations have been published (see <a href="#deliverables">list of deliverables below</a>), 
+          the Working Group will continue in maintenance mode to handle errata. No new Recommendations are planned. 
+          <a href="https://www.w3.org/2023/Process-20231103/#class-4">Class 4 changes</a> for these Recommendations 
+          are <strong>out of scope</strong>, <em><strong>except</strong></em>: 
         </p>
+
+        <ul>
+          <li>
+            For features that had been listed as "At Risk" during the CR phase of the Verifiable Data Model 
+            specification, had been removed from the final document due to missing the CR Exit Criteria, but 
+            have been developed further by the community providing a possibly adequate consensus and support 
+            to be added to the Recommendation. At the time of writing this charter, these terms are: 
```

Comment:
```suggestion
            For features that had been listed as "At Risk" during the CR phase of the Verifiable Data Model 
            specification, had been removed from the final document due to missing the CR Exit Criteria, but 
            have been developed further by the community providing a possibly adequate consensus and support 
            to be added to the Recommendation. At the time of writing this charter, these terms are: 
```

This section seems to be missing the predicate that says that "such features shall be in scope" or however the language might go.

Reply:
I see it now.

Confusion resolved.